relay/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM gcr.io/distroless/base:debug
```

Comment:
I suggest we also run this as non-root, i.e.

```
FROM gcr.io/distroless/base

COPY netbird-relay /go/bin/netbird-relay
USER 1000:1000
ENTRYPOINT [ "/go/bin/netbird-relay" ]
```

We might need to change the default port from 443 to something else or document that we need to ensure cap `NET_BIND_SERVICE` is provided (nowadays default for **docker**: https://github.com/moby/moby/pull/41030)